tests/models/oneformer/test_processor_oneformer.py

```diff
@@ -0,0 +1,868 @@
+# coding=utf-8
```

Comment:
Please also add a test file for the image processor

Reply:
Is that necessary? If you look at `test_processor_oneformer.py`, it covers all the tests for the image processor as well. Because both the task_inputs and images are required to run the model, I don't think it makes sense to have one only for image processing.

Reply:
We require this for each model, so yes. People can still use the image processor/tokenizer separately if they prefer so